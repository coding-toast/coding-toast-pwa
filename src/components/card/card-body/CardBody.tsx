import * as React from 'react'
import CardBodyStyles from './CardBody.module.css'

const CardBody: React.FC = ({ children }) => <article className={CardBodyStyles.card}>{children}</article>

export default CardBody
