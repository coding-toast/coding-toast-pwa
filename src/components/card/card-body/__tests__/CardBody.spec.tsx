import React from 'react'
import renderer from 'react-test-renderer'

import CardBody from '../CardBody'

describe('<CardBody />', () => {
  it('renders correctly without children', () => {
    const tree = renderer.create(<CardBody />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('renders with children', () => {
    const tree = renderer
      .create(
        <CardBody>
          <p>Content</p>
        </CardBody>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
