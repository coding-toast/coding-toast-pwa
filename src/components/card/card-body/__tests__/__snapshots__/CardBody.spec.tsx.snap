// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardBody /> renders correctly without children 1`] = `
<article
  className="card"
/>
`;

exports[`<CardBody /> renders with children 1`] = `
<article
  className="card"
>
  <p>
    Content
  </p>
</article>
`;
